src/services/balancer/subgraph/types.ts

```diff
@@ -191,13 +191,20 @@ export interface PoolActivity {
   type: PoolActivityType;
 }
 
+export interface PoolSwapUserAddress {
+  id: string;
+}
+
 export interface PoolSwap {
   tokenIn: string;
   tokenOut: string;
   tokenAmountIn: string;
   tokenAmountOut: string;
   timestamp: number;
   tx: string;
+  userAddress: PoolSwapUserAddress;
```

Comment:
I don't think we need a new interface just for this, we can simplify a little bit:
```suggestion
  userAddress: {
    id: string;
  };
```

Reply:
fix